api/src/main/java/org/openmrs/module/esaudemetadata/MetadataBundleUtils.java

```diff
@@ -6,6 +6,21 @@
 		public static final String WRITE_TEST_ORDER = "Write Test Order";
 		public static final String READ_TEST_ORDER = "Read Test Order";
 		public static final String UPDATE_TEST_ORDER = "Update Test Order";
+		public static final String DELETE_VITALS = "Delete Vitals";
+		public static final String EDIT_VITALS = "Edit Vitals";
+		public static final String READ_ANAMNESIS = "Read Anamnesis";
+		public static final String READ_DIAGNOSIS = "Read Diagnosis";
+		public static final String READ_OMS_STADIUM = "Read OMS Stadium";
+		public static final String READ_PATIENT = "Read Patient";
+		public static final String READ_DRUG_PRESCRIPTION = "Read Drug Prescription";
+		public static final String READ_RELEVANT_ASPECTS = "Read Relevant Aspects";
+		public static final String READ_SOCIAL = "Read Social";
```

Comment:
View social Information? or View social form? etc

Reply:
@ningosi the privilege names come from https://docs.google.com/spreadsheets/d/1HJioc0wpZUsjjIkVM3t_vpb5UJkTtJ1V1i-VOdA_Px0/edit#gid=0